cni-plugin/deployment/scripts/install-cni.sh

```diff
@@ -0,0 +1,222 @@
+#!/bin/sh
+# Copyright (c) 2018 Tigera, Inc. All rights reserved.
+# Copyright 2018 Istio Authors
+# Modifications copyright (c) Nordstrom, Inc
```

Comment:
Our convention is to attribute this copyright to "Linkerd authors" instead of "Nordstrom, Inc". Mind changing it here and in the other places it shows up in this diff?